### DISCUSSION TITLE
SolarFlow Control config.ini smartmeter part

### DISCUSSION THREAD

**Discussion by joe63** [reinhard-brandstaedter/solarflow-control#236] [Q&A]
hello,
my smartmeter measuring tool delivers following mqtt-structure:
![image](https://github.com/reinhard-brandstaedter/solarflow-control/assets/6555368/8c8b2d6a-c3c0-4d34-9d0d-bce2eaf068b8)

how should I input this in the items 1, 2, 3?

![image](https://github.com/reinhard-brandstaedter/solarflow-control/assets/6555368/3a26275d-0a43-47a4-a646-b91d3d45f2df)
 
Thanks and BR
Joe

**Comment by reinhard-brandstaedter** [comment]
Set your base topic to
```feeds/integer/Smartmeter/Wirkenergie/MomentanleistungSaldo```
and leave the rest empty, or on default. They won't be used as the values are already integers/floats

**Reply by joe63** [reply]
Thank you for the quick response
